packages/keystone/crypto/EncryptionManager.js

```diff
@@ -0,0 +1,267 @@
+const crypto = require('crypto')
+
+const isEmpty = require('lodash/isEmpty')
+const isNil = require('lodash/isNil')
+
+const conf = require('@open-condo/config')
+
+const { compressors } = require('./compressors')
```

Comment:
It looks like a global const {[name]: function} it’s better to use consts like naming.

Reply:
It’s better to use this global cost for EncryptionVersion. I know too many cases and usually I need to write own EncryptionVersion …